.nais/kafka/arbeidsuforhet-vurdering.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: kafka.nais.io/v1
+kind: Topic
+metadata:
+  annotations:
+    dcat.data.nav.no/title: "Vurderinger relatert til arbeidsuforhet for sykmeldte personer"
+    dcat.data.nav.no/description: >-
+      Topic inneholder informasjon om vurderinger relatert til arbeidsuforhet for sykmeldte personer.
+  name: arbeidsuforhet-vurdering
+  namespace: teamsykefravr
+  labels:
+    team: teamsykefravr
+spec:
+  pool: {{ kafkaPool }}
+  config:
+    cleanupPolicy: delete
+    minimumInSyncReplicas: 1
+    partitions: 4
+    replication: 3
+    retentionBytes: -1  # -1 means unlimited
+    retentionHours: -1  # -1 means unlimited
+  acl:
+    - team: teamsykefravr
+      application: isarbeidsuforhet
+      access: readwrite
+    - team: disykefravar
+      application: dvh-sykefravar-airflow-kafka
+      access: read
```

Comment:
Kan godt gi lese-tilgang til ispersonoppgave her også siden den skal konsumere vurderingene og behandle oppgaven for å vurdere avslag etter utløpt forhåndsvarsel.